src/contract.js

```diff
@@ -55,7 +55,7 @@ function ContractFactory(contractABI, extend) {
                     to: self.address,
                   });
 
-                methodTxObject.data = encodeMethodReadable(methodObject, methodArgs)
```

Comment:
@zachferland we can still consume stuff from them. We still support the data field. But function overrides it